### DISCUSSION TITLE
Standup (Friday, May 6, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#46] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
 - I finished my course

**What are your priorities for the next 24 hours?**:
 - do what you make me do

**Blockers**:
 -

**Shoutouts**:
 - @CarlosCRG19 @CristianCazares @BrandonMagana @DavBot02 @antonieto

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Material UI stuff
 - Started de styling part of a form based on the use case

**What are your priorities for the next 24 hours?**:
 - Finished said form completely


**Shoutouts**:
 - @everyone for being cooperative

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Implemented Company entity 
 - Finished first endpoints and stablished communication between db and server

**What are your priorities for the next 24 hours?**:
 -  Start  implementing other entities. Eg: job aspirant

**Blockers**:
 -None

**Shoutouts**:
 - @antonieto  for yesterday's productive marathon

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on Figma wireframes
 - Define a Theme

**What are your priorities for the next 24 hours?**:
 - Finish wireframes

**Blockers**:
 - None

**Shoutouts**:
 - @BrandonMagana @antonieto for their work on backend

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Still reading about Material UI
 - Working on "create job offer" view

**What are your priorities for the next 24 hours?**:
 - Finish view

**Blockers**:
 -

**Shoutouts**:
 - @everyone

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished the first CRUD endpoints for JobOffer resource 
 - Created a Postman workspace to collaborate 
 
**What are your priorities for the next 24 hours?**:
 - Read more about auth0 
 - (maybe) create our first authentication middleware 
 - (maybe) define auth roles using auth0

**Blockers**:
 - None lol

**Shoutouts**:
 - @BrandonMagana for writing a ton of great code yesterday :D
